### ISSUE TITLE
Organizar los experimentos

### ISSUE THREAD

**Issue by facundoq** [midusi/handshape-classification#1]
* **Crear una carpeta de resultados
* Crear una clase Experimento. La subclasifican por cada experimento que hacen. 
* Cada experimento tiene un id. Los Ids son únicos. Cuando se ejecuta un experimento, se crea su directorio en base a su ID, dentro de la carpeta de resultados.
* Cada experimento puede ser una combinación de modelo y de base de datos. Hacer clases abstractas para no repetir código
* Cuando termina un experimento, guarda sus resultados en su subdirectorio. Guardar los gráficos de loss accuracy, un txt con los parámetros, y el resultado final.**
* Tener un script que "corra" una lista de objetos experimento.

**Comment by facundoq** [comment]
Matriz de confusión:
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html

https://scikit-learn.org/stable/auto_examples/model_selection/plot_confusion_matrix.html
